cfgov/legacy/templates/hud/housing_counselor.html

```diff
@@ -97,11 +97,7 @@ <h1>Find a housing counselor</h1>
                             </div>
                             <div id="hud_hca_api_more_info_container" class="hud_hca_api_more_info">
                                 <p>
-                                    This tool is open sourced on
-                                    <a href="https://github.com/cfpb/django-hud">GitHub</a>
-                                    and powered by <a href="https://data.hud.gov/housing_counseling.html">HUD's</a>
-                                    official list of housing counselors.
-                                    We encourage you to leverage it in your own applications.
+                                    This tool is powered by <a href="https://data.hud.gov/housing_counseling.html">HUD's</a> official list of housing counselors.
```

Comment:
One minor request: see comment by @schaferjh with modified text on the GHE issue. There should be another sentence here.

Reply:
@chosak The other sentence she included is the next paragraph, which remains unchanged.

Reply:
@chosak @richaagarwal I am actually agnostic -- leaving it as two paragraphs is fine.